fwdpy11/discrete_demography.py

```diff
@@ -535,3 +525,16 @@ def _timed_events(self):
             self.set_migration_rates,
         ]:
             yield i
+
+
+def from_demes(
```

Comment:
To keep the API simpler, I think we should import this function into fwdpy11.dempgraphy, so that it is called as `fwdpy11.dempgraphy.from_demes` as opposed to `fwdpy11.demography.discrete_demography.from_demes`?

Reply:
nevermind--I wasn't thinking straight!